internal/pkg/test/cmd.go

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package test
+
+import (
+	"testing"
+
+	"gotest.tools/icmd"
+)
+
+type Cmd struct {
+	path string
+}
+
+type Result struct {
+	*icmd.Result
+}
+
+func NewCmd(path string) *Cmd {
+	c := &Cmd{
```

Comment:
this can be return `&Cmd{ path: path }`, no need to create the variable to return it on the next line.

Reply:
+1

Reply:
Done. I thought I would add more arguments but perhaps that will be later.